src/assignments/ray/lesson_2_answers.clj

```diff
@@ -11,6 +11,23 @@
 
 (toS words_1)
 
+(defn che [words]
+  (group-by frequencies words))
+
+(che words_1)
+
+(defn into-set-1 [words]
+  (let [m (group-by frequencies words)]
+    (into #{} (vals m))))
+
+(into-set-1 words_1)
+
+(defn into-set-2 [words]
```

Comment:
The functions into-set-1 and into-set-2 do something wildly different; can you see why those might be poor names for these functions?